pkgs/applications/science/electronics/linux-gpib/linux-gpib-user/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchurl
+, autoconf
+, libtool
+, bison
+, flex
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib-user";
+  version = "4.3.6";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/linux-gpib/files/linux-gpib%20for%203.x.x%20and%202.6.x%20kernels/${version}/linux-gpib-${version}.tar.gz";
+    hash = "sha256-Gze4xrvkhEgn+J5Jhrycezjp2uhlD1v6aX0WGv4J2Jg=";
+  };
+
+  unpackPhase = ''
+    tar xf $src
+    tar xf linux-gpib-${version}/${pname}-${version}.tar.gz
+  '';
```

Comment:
Is that really double packed?

Reply:
Yes. The main tarball consists of two nested tarballs, containing the package for the kernel and the user.